lux/action/correlation.py

```diff
@@ -51,6 +51,9 @@ def correlation(ldf: LuxDataFrame, ignore_transpose: bool = True):
     recommendation = {
         "action": "Correlation",
         "description": "Show relationships between two <p class='highlight-descriptor'>quantitative</p> attributes.",
+        "longDescription": "Correlation searches through all pairwise relationship between two quantitative attributes \
+            (e.g., age, capital-gain). The visualizations are ranked from most to least linearly correlated based on \
```

Comment:
`age, capital-gain` needs to be adaptive here

Reply:
resolved

Reply:
I don't think this is adaptive yet?